optaweb-vehicle-routing-docs/src/main/asciidoc/index.adoc

```diff
@@ -1 +1,47 @@
-= OptaWeb Vehicle Routing User Guide
+[id="vrp-quick-start"]
+
+= Using OptaWeb Vehicle Routing
+
+:context: vrp-user-guide
+
+As a developer, you can use OptaWeb Vehicle Routing to optimize your vehicle fleet deliveries.  In this guide, you will create and run a sample OptaWeb Vehicle Routing application.
+
+include::modules/con-vrp.adoc[leveloffset=+1]
```

Comment:
Question: What are the prefixes good for? What does `con` mean?

Can we avoid repeating the `vrp` keyword. This whole documentation is about VRP. We would need to use it in every file to be consistent. It doesn't bring and valuable information.

Reply:
I'd prefer if the file name reflected the title of its main section so that it's easy to guess what's inside it. For example this file could be named `con-what-is-vehicle-routing.adoc`.

Reply:
For modular documentation, we use the prefixes so that we as writers can easily identify the type of module the file contains. In addition, the tooling team has recently required us to use the prefixes. The use (or will use) the designation when rendering the docs.

Sure, I can rename the ID, no problem. However, we might change the headings so it makes a little more work to maintain. Also, some of the headings are long. But we can see how it goes.

Reply:
I know what you mean about repeating VRP. The reason that we do it is because soon, in the not too distant future, users will be able to find individual modules when they search. For a module, each page is page one. So when you look at a module, it must make sense out of context of the rest of the document. I know its a different way to look at writing. It's a compromise.